R/robust-lmrob-tidiers.R

```diff
@@ -22,12 +22,35 @@
 #' @export
 #' @family robust tidiers
 #' @seealso [robust::lmRob()]
-#' @include stats-lm-tidiers.R
-tidy.lmRob <- function (x, ...) {
-  dots <- enquos(...)
-  dots$conf.int <- FALSE
-  
-  rlang::exec(tidy.lm, x, !!!dots)
+tidy.lmRob <- function (x, quick = FALSE,...){
+  if (quick) {
+    co <- stats::coef(x)
+    ret <- data.frame(term = names(co), estimate = unname(co), 
+                      stringsAsFactors = FALSE)
+    
+    return(as_tibble(ret))
+  }
+  s <- robust::summary.lmRob(x)
+  ret <- tidy.summary.lmRob(s)
```

Comment:
Instead of having a separate function `tidy.summary.lmRob()` can we do all the processing in `tidy.lmRob()`?